CONTRIBUTING.md

```diff
@@ -35,6 +35,24 @@ By contributing to this project you agree to the Developer Certificate of Origin
 statement that you, as a contributor, have the legal right to make the
 contribution. See the [DCO](DCO) file for details.
 
+### Git commit Sign-off
+
+This line should be included at the bottom of git commit message.
+
```

Comment:
Can you please rewrite the line? You can have something like this or just rephrase it: “Commit message should contain signed off section with full name and email. For example:”

Reply:
Otherwise it sounds like John Doe should be the fixed name

Reply:
I will change it.